app/javascript/components/mentoring/Queue.tsx

```diff
@@ -136,6 +130,8 @@ export const Queue = ({
           isFetching={isTrackListFetching}
           value={selectedTrack}
           setValue={setSelectedTrack}
+          cacheKey={TRACKS_LIST_CACHE_KEY}
```

Comment:
Does this work correctly if you login, view this page (cache is created), logout, login as a different user and visit this page again? In other words: is this tied to the current user?

Reply:
It is not scoped per user. if the whole process (logging in and out) happens in less than 5 minutes, the cached values will be used.